sdk/communication/azure-communication-callingserver/azure/communication/callingserver/_converters/_converter.py

```diff
@@ -311,3 +313,35 @@ def convert(
             target_participant=target_participant,
             user_to_user_information=user_to_user_information
         )
+
+class StartCallRecordingRequestConverter(object):
+    @staticmethod
+    def convert(
+        recording_state_callback_uri # type: str
+        ): # type: (...) -> StartCallRecordingRequest
+
+        if not recording_state_callback_uri:
+            raise ValueError("recording_state_callback_uri can not be None")
+
+        return StartCallRecordingRequest(
+            recording_state_callback_uri=recording_state_callback_uri
+        )
+
+class StartCallRecordingWithCallLocatorRequestConverter(object):
+    @staticmethod
+    def convert(
+        call_locator, # type: CallLocatorModel
+        recording_state_callback_uri # type: str
+        ): # type: (...) -> StartCallRecordingWithCallLocatorRequest
+
+        if not call_locator:
+            raise ValueError("call_locator can not be None")
+        if not recording_state_callback_uri:
+            raise ValueError("start_call_recording_request can not be None")
+
+        return StartCallRecordingWithCallLocatorRequest(
+            call_locator=call_locator,
+            start_call_recording_request=StartCallRecordingRequestConverter.convert(
```

Comment:
same, why do we need to do this call if we could just create the request.

Reply:
@zihzhan-msft

Reply:
I agree, we should remove the converter for these one line request.

Reply:
Removed convertor